src/components/site-header/SiteHeaderLogos.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import edXLogo from '@edx/brand/logo.svg';
+
+import PropTypes from 'prop-types';
+import getsmarterLogo from '../../assets/icons/getsmarter-logo.svg';
+
+const SiteHeaderLogos = ({ enterpriseConfig }) => {
+  const { pathname } = useLocation();
+
+  let mainLogo = (
+    <div className="d-flex">
+      <img
+        className="d-block logo"
+        src={enterpriseConfig.branding.logo || edXLogo}
+        alt={`${enterpriseConfig.name} logo`}
+        data-testid="header-logo-image-id"
+      />
+    </div>
+  );
+
+  if (!enterpriseConfig.disableSearch) {
+    mainLogo = (
+      <Link to={`/${enterpriseConfig.slug}`} data-testid="header-logo-link-id">
+        {mainLogo}
+      </Link>
+    );
+  }
+
+  return (
+    <>
+      {mainLogo}
+      {pathname.includes('executive-education-2u') && (
+        <img
+          className="d-block getsmarter-logo ml-1"
```

Comment:
[curious] The `.logo` class used on `mainLogo` has a `margin-right` property; just curious about the interaction between the existing `margin-right` on `.logo` and the `ml-1` on the GetSmarter logo.

Reply:
The margin was only applied on the desktop view. I updated it so it's applied in both mobile & desktop views and got rid of the ml-1.